Webapp/sources/services/queries/workgroup.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+from sources import models
+from sources.extensions import db
+
+
+def get_all_workgroups() -> List[models.WorkGroup]:
```

Comment:
the nice thing about moving things into services, is now the domain can better describe what the function is doing as well, so we can make things simpler.

this for example could be called `list()` now, as it's in the workgroup service. 
obviously renaming is low priority too
https://journal.stuffwithstuff.com/2009/06/05/naming-things-in-code/ is always a nice read.